api/types.go

```diff
@@ -91,6 +99,7 @@ type Options struct {
 	NumKeep          int      `json:"num_keep,omitempty"`
 	Seed             int      `json:"seed,omitempty"`
 	NumPredict       int      `json:"num_predict,omitempty"`
+	NProbs 		     int      `json:"n_probs,omitempty"`
```

Comment:
Hi, Your code is not formatted, please run below commang to format code. and `git add`, `git commit` again
```shell
gofmt -w api/types.go
```

Reply:
done